license-metadata.json

```diff
@@ -205,53 +161,6 @@
             },
             "name": "library/std/src/sys/sync/mutex/fuchsia.rs",
             "type": "file"
-          },
-          {
-            "children": [],
-            "license": {
-              "copyright": [
-                "Rust on Embedded Devices Working Group",
-                "The Rust Project Developers (see https://thanks.rust-lang.org)"
-              ],
-              "spdx": "Apache-2.0 OR CC-BY-SA-4.0 OR MIT"
-            },
-            "name": "src/doc/embedded-book",
```

Comment:
I assume it's because you've got your config.toml set up to avoid building the embedded book and gcc, so they don't wind up included in this file even though they should be.

To fix this, would either do a truly complete rust build, or I would just hand-patch in the few lines that are added in your PR.

Reply:
Ok, that makes more sense now. Thanks a lot!